src/test/java/definitions/LoginDefinitions.java

```diff
@@ -0,0 +1,33 @@
+package definitions;
+
+import com.report.portal.business.steps.BaseSteps;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class LoginDefinitions {
+
+    private final BaseSteps baseSteps = new BaseSteps();
+
+    @Given("Login page is open")
+    public void loginPageIsOpen() {
+        baseSteps.openInitialPage();
+    }
+
+    @When("User enters valid credentials")
+    public void userEntersValidCredentials() {
+        baseSteps.login();
+    }
+
+    @Then("User is signed in")
+    public void userIsSignedIn() {
+        baseSteps.isLogoVisible();
```

Comment:
On the definitions level it is better to explicitly use assertions or use unambiguously "assertive" methods. Like
```Assertions.assertThat(baseSteps.assertLogoIsVisible()).isTrue()```
or
```baseSteps.assertLogoIsVisible()```

Reply:
I decided to use Selenide assertions in my tests. should I refactor the whole project to use testng assertions?

Reply:
It is up to you.
But it is better to centralize your assertions: maybe you will have RestAssured with its included response validators for http actions, then another library...
As a result you will obtain a mess of assertions in different format and loose effective control over them.